### ISSUE TITLE
get returns empty structs

### ISSUE THREAD

**Issue by dnbx** [JimHokanson/labchart_server_matlab#1]
Hello Jim,

I am trying to stream data from labchart to MATLAB, but after creating 

a=actxserver('ADIChart.Document');
b=actxGetRunningServer('ADIChart.Application');

get(a) and get(b) return empty "structs with no fields". How can I access handles like "ActiveDocument"?

Best,
B 

P.S. Thank you for making your code accessible!

**Comment by dnbx** [comment]
Installing LabChart 8 solved this problem (no license). Still works even though I am using LabChart 7.3.8.

**Comment by JimHokanson** [comment]
I'm glad you were able to get things working. Thanks for the info. It may be helpful to someone else if they run into the same issue.

Note, I started work on a streaming interface although I never really used it. I created an example for a friend. The example code can be found here:

https://github.com/JimHokanson/labchart_server_matlab/blob/master/%2Blabchart/%2Bstreaming/%40streamed_data1/streamed_data1.m

There is an example in the documentation. It also helps to read the optional inputs in the constructor (streamed_data1 method) that currently starts around line 184.
